platforms/linux-x64/external.sh

```diff
@@ -165,6 +165,7 @@ if [ "${BGFX_EXPECTED_SHA}" != "${BGFX_FOUND_SHA}" ]; then
       -DBGFX_BUILD_EXAMPLES=OFF \
       -DBGFX_CONFIG_MULTITHREADED=ON \
       -DBGFX_CONFIG_MAX_FRAME_BUFFERS=256 \
+      -DBGFX_WITH_WAYLAND=ON \
```

Comment:
I don't think this is required as this is set automatically on linux is my understanding of below linked line is correct.

see https://github.com/bkaradzic/bgfx.cmake/blob/36a0d1d8d040113a954f99b1456121072dc77348/CMakeLists.txt#L46